api/src/main/java/org/apache/cloudstack/api/response/GuestOSResponse.java

```diff
@@ -35,13 +35,17 @@ public class GuestOSResponse extends BaseResponse {
     @Param(description = "the ID of the OS category")
     private String osCategoryId;
 
-    @SerializedName(ApiConstants.DESCRIPTION)
-    @Param(description = "the name/description of the OS type")
-    private String description;
```

Comment:
for backwards compatibility, would it be better to keep "description" ?
it can be marked as deprecated.